python/doc/examples/functional_modeling/vectorial_functions/plot_functions_inputDim.py

```diff
@@ -0,0 +1,136 @@
+"""
+==========================================
+Increase the input dimension of a function
+==========================================
+"""
+# %%
+# Import the required modules
+
+# %%
+import openturns as ot
+
+# %%
+# Description
+# ===========
+#
+# We want to build a function :math:`f : \mathbb{R}^d \mapsto \mathbb{R}` from *p* functions :math:`f_i: \mathbb{R} \mapsto \mathbb{R}`.
+#
+# We can do that:
+#
+#     - Case 1: by tensorisation of the functions :math:`f_i`,
+#     - Case 2: by addition of the functions :math:`f_i`.
+#
```

Comment:
This produces a blank space:
![image](https://user-images.githubusercontent.com/31351465/230656085-ae0b9c2d-a316-4ea2-b7cd-4275ecfc3ecb.png)

Reply:
ok